1_prep.R

```diff
@@ -59,9 +50,32 @@ p1 <- list(
     p1_nldas_dates,
     munge_nldas_dates(p1_nldas_csvs, p1_nldas_time_period)),
 
+
+  # Prep observed data for calibration ------------------
+
+  # Pull in observed data from `lake-temperature-model-prep``
+  tar_target(p1_merged_temp_data_daily_feather,
+             '1_prep/in/merged_temp_data_daily.feather',
+             format = 'file'),
+
+  # create an RDS file for each MO model for calibration
+  tar_target(p1_obs_rds,
+             subset_model_obs_data(data = p1_merged_temp_data_daily_feather,
+                                   nhdhr_id = p1_nldas_site_ids,
+                                   path_out = '1_prep/out'),
+             pattern = map(p1_nldas_site_ids),
+             format = 'file'
+  ),
+
+  # model config and set up------------------------------
+
   # Set up NLDAS model config
   tar_target(p1_nldas_model_config,
-             build_nldas_model_config(p1_nldas_site_ids, p1_nldas_nml_list_subset, p1_nldas_csvs, p1_nldas_dates)
+             build_nldas_model_config(p1_nldas_site_ids,
+                                      p1_nldas_nml_list_subset,
+                                      p1_nldas_csvs,
+                                      p1_obs_rds,
+                                      p1_nldas_dates)
```

Comment:
`p1_nldas_dates` is a holdover from Hayley's work. It's a one row tibble that contains a bunch of time info: driver dates, burn in/burn out, etc. I think it does need to be created somewhere because that info is part of the `p1_nldas_model_config` tibble. The two choices for "where" are : (1) as it's own target [the current set up], or (2) within the `build_nldas_model_config` function. 

My inclination is to keep it "as is" because it isn't expensive and it cuts down on the amount of "stuff" happening in `build_nldas_model_config()`.

All this being said, if we are looking for targets to remove, it seems to me that `p1_nldas_time_period` is ripe for removal. It is a [user-specified target](https://github.com/padilla410/lake-temperature-missouri-models/blob/6dcd8f520ad46e3382eb8152fdd79b560b3865ec/1_prep.R#L36-L37). I could just as easily specify it as an argument in `munge_meteo()` which is the function used to create `p1_nldas_dates`.

Reply:
...giving this a little more thought... I think you're right. This duplicate pathway is not necessary:
![image](https://user-images.githubusercontent.com/15007294/177596284-0cc54fc9-2539-4acd-89be-2a4af43e8716.png)

I am going to move the generation of `p1_nldas_dates` inside of `build_nldas_model_config()`